### ISSUE TITLE
issue about cross domain 

### ISSUE THREAD

**Issue by finger-monkey** [JDAI-CV/fast-reid#132]
“We propose a cross-domain method FastReID-MLT that adopts mixture label transport to learn pseudo
label by multi-granularity strategy. We first train a model with a source-domain dataset and then finetune on the pre-trained model with pseudo labels of the target-domain dataset.”我们提出了一种跨域方法FastReID-采用混合标签传输的MLT通过多粒度策略学习伪标签。我们首先使用源域数据集训练一个模型，然后使用目标域数据集的伪标签对预训练的模型进行微调。
不知道这个具体是如何实现的？

**Comment by L1aoXingyu** [comment]
This paper is under review, we will release all codes after the result comes out.
